akka-actor/src/main/resources/reference.conf

```diff
@@ -590,6 +590,12 @@ akka {
     # you can turn this off.
     warn-about-java-serializer-usage = on
 
+    # To be used with the above warn-about-java-serializer-usage
+    # When warn-about-java-serializer-usage = on, and this suppress-warning-on-no-verification = on,
+    # warnings are suppressed for classes extending NoSerializationVerificationNeeded
+    # to reduce noize.
+    suppress-warning-on-no-verification = off
```

Comment:
Sure, but actually let me use "warn-on-no-serialization-verification" (i.e. remove "suppress") to make it shorter.